strings/base_error.h

```diff
@@ -560,7 +560,7 @@ WINRT_EXPORT namespace winrt
         return pointer;
     }
 
-    [[noreturn]] inline void terminate() noexcept
+    inline void terminate() noexcept
```

Comment:
Really? This one is surprising to me.

Reply:
Yes, Clang can't tell that the function won't return and thus rejects the [[noreturn]]. Unfortunately, there seems to be no way to mark a lambda as [[noreturn]].